lexicons/app/bsky/unspecced/checkHandleAvailability.json

```diff
@@ -0,0 +1,74 @@
+{
+  "lexicon": 1,
+  "id": "app.bsky.unspecced.checkHandleAvailability",
+  "defs": {
+    "main": {
+      "type": "query",
+      "description": "Checks whether the provided handle is available. If the handle is not available, available suggestions will be returned. Optional inputs will be used to generate suggestions.",
+      "parameters": {
+        "type": "params",
+        "required": ["handle"],
+        "properties": {
+          "handle": {
+            "type": "string",
+            "format": "handle",
+            "description": "Tentative handle. Will be checked for availability or used to build handle suggestions."
+          },
+          "email": {
+            "type": "string",
+            "description": "User-provided email. Might be used to build handle suggestions."
+          },
+          "birthDate": {
+            "type": "string",
+            "format": "datetime",
+            "description": "User-provided birth date. Might be used to build handle suggestions."
+          }
```

Comment:
Maybe worth mentioning that this data is not stored or transmit via this endpoint, will only be used for suggestions.

Reply:
Talked offline, this should be obvious from the code, no real need to specify here 👍